src/__tests__/__snapshots__/object-props-test.js.snap

```diff
@@ -3,5 +3,6 @@ exports[`test object-props 1`] = `
 
 var Component = function Component(props) {
   return React.createElement(\"div\", null);
-};"
+};
+Component.propTypes = require(\"prop-types\").object;"
```

Comment:
@brigand I agree about C.proptypes being an object; however, is this usage here OK?